universum.cabal

```diff
@@ -71,10 +72,11 @@ library
     safe                  >= 0.3    && <0.4,
     stm                   >= 2.4    && <2.5,
     text                  >= 1.2    && <1.3,
-    utf8-string           >= 1.0    && <1.1,
     text-format,
     transformers          >= 0.4    && <0.6,
+    type-operators        == 0.1.0.4,
```

Comment:
In earlier versions they had no `<+>` and `<=>` operators.
On the other side, after `0.1.0.0` they removed `%` operator. Looks like author doesn't care much about backward compatibility